src/Compilers/CSharp/Portable/LanguageVersion.cs

```diff
@@ -80,6 +80,11 @@ public enum LanguageVersion
         CSharp7_2 = 702,
 
         /// <summary>
+        /// C# language version 8.
+        /// </summary>
+        CSharp8 = 800,
```

Comment:
>800 [](start = 18, length = 3)

Should it be 8 instead?

Reply:
800 is correct. We need the values in this enum to be ordered.